turn/src/allocation/allocation_manager.rs

```diff
@@ -13,13 +13,15 @@ use util::Conn;
 // ManagerConfig a bag of config params for Manager.
 pub struct ManagerConfig {
     pub relay_addr_generator: Box<dyn RelayAddressGenerator + Send + Sync>,
+    pub gather_metrics: bool,
```

Comment:
I think we should consider using a feature instead of a bool for this

Reply:
I do not think so. [Comments say](https://github.com/logist322/webrtc/blob/98d7a5754e8022d71418d672bcfe85ac34b6f187/turn/src/server/config.rs#L32) different connections can have custom behavior: 

```
// conn_configs are a list of all the turn listeners
// Each listener can have custom behavior around the creation of Relays
pub conn_configs: Vec<ConnConfig>,
```

Do you think we should have the same settings for every **Allocation** `Manager` on the `Server` instance?

Reply:
All right, so it will be a feature.